galley/pkg/source/kube/builtin/testdata/node.yaml

```diff
@@ -131,7 +131,7 @@ status:
     sizeBytes: 446398900
   - names:
     - gcr.io/istio-release/proxyv2@sha256:dec972eab4f46c974feec1563ea484ad4995edf55ea91d42e148c5db04b3d4d2
-    - gcr.io/istio-release/proxyv2:master-latest-daily
+    - gcr.io/istio-release/proxyv2:release-1.3-latest-daily
```

Comment:
nit - would rather we just leave these or make them something arbitrary like `tag` so we don't have to keep changing them. The tag here is meaningless, its just test data

Reply:
although I guess that would have to be a change on master